pegasus/sites.v3/code.org/views/professional_learning_skinny_banner.haml

```diff
@@ -0,0 +1,23 @@
+- display_professional_learning_banners = true
+- if request.locale == "en-US" && display_professional_learning_banners
+  = inline_css 'professional_learning_marketing_banner.css'
+  - header_text = 'Help students become superheroes!'
```

Comment:
I know this is probably out of scope, but I'd love for even US-only content to be translated. Not required!

Reply:
Good call — I'll update!

Reply:
🎉